beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -51,6 +51,11 @@ func (r *RegularSync) validateBeaconBlockPubSub(ctx context.Context, msg proto.M
 		return false, err
 	}
 
+	if r.chain.FinalizedCheckpt().Epoch > helpers.SlotToEpoch(m.Slot) {
+		log.Debug("Block older than finalized checkpoint received,rejecting it")
```

Comment:
```suggestion
		log.Debugf("Rejecting block older than finalized checkpoint epoch, %d > %d", r.chain.FinalizedCheckpt().Epoch, helpers.SlotToEpoch(m.Slot))
```

Reply:
Actually can we use withField? it's easier to query in elastic search if there is no formatting in the message name

Reply:
Good point